app/renderer/components/navbar.js

```diff
@@ -1,3 +1,4 @@
+/* globals $ */
```

Comment:
Please, avoid this, move the method ` $(".tap-target").tapTarget('close');` to the file `materialize.js`, wrap it and use that in this class

If we need to add crappy code because of materialize, at least lets get it all in the same place :)